tests/foreman/api/test_subnet.py

```diff
@@ -278,6 +277,52 @@ def test_positive_subnet_parameters_override_impact_on_subnet(self):
         CaseLevel: System
         """
 
+        # Create subnet with valid parameters
+        parameter = [{'name': gen_string('alpha'), 'value': gen_string('alpha')}]
+        org = entities.Organization().create()
+        loc = entities.Location(organization=[org]).create()
+        org_subnet = entities.Subnet(
+            location=[loc],
+            organization=[org],
+            subnet_parameters_attributes=parameter,
+        ).create()
+        self.assertEqual(
+            org_subnet.subnet_parameters_attributes[0]['name'],
+            parameter[0]['name']
+        )
+        self.assertEqual(
+            org_subnet.subnet_parameters_attributes[0]['value'],
+            parameter[0]['value']
+        )
+        # Create host with above subnet
+        host = entities.Host(
+            location=loc,
+            organization=org,
+            subnet=org_subnet,
+        ).create()
+        self.assertEqual(host.subnet.read().name, org_subnet.name)
+        new_parameter = [{
+            'name': '{}'.format(org_subnet.subnet_parameters_attributes[0]['name']),
```

Comment:
`'{}'.format(org_subnet.subnet_parameters_attributes[0]['name'])`?
That operation does nothing, so please use variable directly

Reply:
Done.